client/src/components/QuestionsAndAnswers.jsx

```diff
@@ -1,8 +1,49 @@
 import React from 'react';
 
 const QuestionsAndAnswers = () => {
+
   return (
-    <h1>Questions And Answers</h1>
+    <div id="qa-div">
+      <h1 id="qa-header">QUESTIONS AND ANSWERS</h1>
+
+      <form id="search">
+        <input className="qa-searchbar" placeholder="HAVE A QUESTION? SEARCH FOR ANSWERS..."></input>
+        {/* NOTE - Search button will be replaced with icon later on  */}
+        <button className="qa-search-btn">Search</button>
+      </form>
+
+      <p></p>
+
+      {/* NOTE - Refactor to have a seperate component later on when there is more to display */}
+      <div className="qa-card-sample">
+        <b><div className="qa-div">Q: Question [Placeholder]</div></b><p></p>
+        <div className="qa-div"><b>A: </b>Answer [Placeholder]</div><p></p>
+        <div className="qa-footer">
+          <label className="qa-username">by Username [Placeholder],
+            <label className="qa-date"> Date Pladeholder
+              <label className="qa-helpful"> Helpful? [Placeholder]
+                <label className="qa-report"> Report [Placeholder] <p></p>
+                  <label className="qa-load-more"><b>[LOAD MORE ANSWERS]</b>
+                  </label>
+                </label>
+              </label>
+            </label>
+          </label>
+        </div>
+      </div>
+
+
+      <p></p>
```

Comment:
Are these p's just for spacing purpose rn?

Reply:
Yeah, spacing for now! Figured we'll need to re-do the css to all match at some point so didn't want to play around with it too much for now.